src/service/chat.js

```diff
@@ -132,11 +132,11 @@ const sendAnonymousDMOtherProfile = async ({
 
 const sendSignedDMOtherProfile = async ({user_id, message}) => {
   const payload = {
-    members: [user_id],
+    member: user_id,
     message
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the payload for the `api.post` method has been changed from an array of members to a single member. Ensure that the backend API has been updated to handle this change.
```diff
-    members: [user_id],
+    member: user_id,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->